src/content/docs/en/guides/cms/builderio.mdx

```diff
@@ -3,19 +3,505 @@ title: Builder.io & Astro
 description: Add content to your Astro project using Builder.io’s visual CMS
 type: cms
 service: Builder.io
-stub: true
+stub: false
 i18nReady: false
 ---
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import FileTree from '~/components/FileTree.astro';
 
+[Builder.io](https://www.builder.io/) is a visual CMS that supports drag-and-drop content editing for building websites. 
 
+At the end of this recipe, you'll be able to create blog posts in the Builder app and have your Astro site update with this new content automatically using a webhook.
+
+To do this, you'll use Builder's [Content API](https://www.builder.io/c/docs/content-api) and [Qwik API](https://www.builder.io/c/docs/qwik-api) to connect your Builder space to Astro with zero client-side JavaScript.
```

Comment:
```suggestion
This recipe will show you how to connect your Builder space to Astro with zero client-side JavaScript.
```
Ugh, some of my suggestions did NOT apply. Sorry for the little paper cuts here as I change some things manually!